File: C/lt_01_estseq/lt01_estseq_18.c
#include <stdio.h>

int difer(int num1, int num2) {

  if (num1 > num2) {

    int aux = num1;
    num1 = num2;
    num2 = aux;
  }
  return num2 - num1;
}

int main() {

  int num1, num2;
  for (int i = 0; i < 2; i++) {

    printf("Insira um número Inteiro: ");
    if (i == 1) {
      scanf("%d", &num1);
    } else {
      scanf("%d", &num2);
    }
  }
  printf("O valor da diferença e %d.\n", difer(num1, num2));
  return 0;
}

File: Python/lt_01_estseq/lt01_estseq_18.py
num1=float(input("Insira o 1º Valor: "))
num2=float(input("Insira o 2º Valor: "))
if num2>num1:
    aux=num1
    num1=num2
    num2=aux
print("O valor da diferença é",num1-num2,".")
